dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/ArtifactDownloader.java

```diff
@@ -79,9 +81,6 @@ private Future<String> submitDownloadRequest(String coords, String fileType, Str
 
     public Set<String> getMissingFeaturesFromRepo(List<String> mavenCoords, MavenRepository repository) throws InstallException {
         info(Messages.INSTALL_KERNEL_MESSAGES.getLogMessage("STATE_CONTACTING_MAVEN_REPO"));
-        checkValidProxy();
```

Comment:
Authentications will be configured in line 92 (when we call `testConnection(repository)`).